templates/content/page-title.html.twig

```diff
@@ -17,7 +17,7 @@
 {% if title %}
   {% include "@atoms/02-text/00-headings/heading-1/heading-1.twig"
     with {
-      "heading": title,
+      "heading": title
```

Comment:
We should keep the `,` on every line (including only lines, and last lines) so that PRs only show new lines of code and don't report this line as changed when we add a comma because we've added a second line.

Wow... that's kinda confusing... if we don't include the comma here, and then add a line to make it
```
"heading": title,
"heading_blockname": "awesome",
```

The diff will show two lines have changed (because of the comma) instead of just showing the one new line as a change.